micrometer-spring-legacy/src/samples/java/io/micrometer/spring/samples/components/okhttp/OkHttpMicrometerInterceptor.java

```diff
@@ -97,4 +82,56 @@ public Response intercept(Chain chain) throws IOException {
 		}
 		return builder;
 	}
+
+	public static class Builder {
+
+
+		private MeterRegistry meterRegistry;
+		private String name;
+		private boolean recordRequestPercentiles = false;
+
+		private Function<Request, Optional<String>> uriMapper = (request) -> Optional.empty();
+		private Function<Response, Optional<String>> statusMapper = (request) -> Optional.empty();
+		private boolean emptyUriIfNoMatch = true;
```

Comment:
Add default tags compare here https://github.com/micrometer-metrics/micrometer/blob/master/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/CaffeineCacheMetrics.java#L102

Reply:
Not really sure what you want me to compare? The tags I have used now are the same as for other outgoing http metrics. What you link to here are cache metrics.

Reply:
Sorry. I was trying to show how other binders add in a default `Iterable<Tag>` that is used and line `78` would change to:

```
List<Tag> tags = Tags.concat(defaultTags,
				"method", request.method(),
				"uri", uri,
				"status", status,
				"clientName", request.url().host()
		);
```